@here/harp-mapview/lib/MapView.ts

```diff
@@ -2827,15 +2828,23 @@ export class MapView extends THREE.EventDispatcher {
                 if (object._backupRenderOrder === undefined) {
                     object._backupRenderOrder = object.renderOrder;
                 }
-                // All objects in a fallback tile must be shifted, such that their renderOrder is
-                // less than the groundPlane that each neighbouring Tile has (it has a renderOrder
-                // of -10000, see addGroundPlane in TileGeometryCreator), only then can we be sure
-                // that nothing of the parent will be rendered on top of the children, as such, we
-                // shift using the FALLBACK_RENDER_ORDER_OFFSET.
-                object.renderOrder = tile.isFallback
-                    ? object._backupRenderOrder - FALLBACK_RENDER_ORDER_OFFSET
-                    : object._backupRenderOrder;
-
+                const isBuilding =
+                    object.userData !== undefined &&
```

Comment:
can't wait for Typescript 3.7!!!

Reply:
yeah, I was just about to use typescript 3.7 but than it came to my mind that we still have 3.6 on the release branch.